File: Server/src/main/java/com/api/server/foodb/MapItemsPathway.java
package com.api.server.foodb;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import jakarta.validation.constraints.Size;
import lombok.Getter;
import lombok.Setter;

import java.time.Instant;

@Getter
@Setter
@Entity
@Table(name = "map_items_pathways")
public class MapItemsPathway {
    @Id
    @Column(name = "id", nullable = false)
    private Integer id;

    @Column(name = "map_item_id")
    private Integer mapItemId;

    @Size(max = 255)
    @Column(name = "map_item_type")
    private String mapItemType;

    @Column(name = "pathway_id")
    private Integer pathwayId;

    @Column(name = "created_at")
    private Instant createdAt;

    @Column(name = "updated_at")
    private Instant updatedAt;

}

File: DotNetServer/DotNetServer/Core/Entities/Foodb/MapItemsPathway.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace DotNetServer.Core.Entities.Foodb;

[Table("map_items_pathways")]
[Index("MapItemId", "MapItemType", Name = "index_map_items_pathways_on_map_item_id_and_map_item_type")]
public class MapItemsPathway
{
    [Key] [Column("id")] public int Id { get; set; }

    [Column("map_item_id")] public int? MapItemId { get; set; }

    [Column("map_item_type")] public string? MapItemType { get; set; }

    [Column("pathway_id")] public int? PathwayId { get; set; }

    [Column("created_at", TypeName = "datetime")]
    public DateTime? CreatedAt { get; set; }

    [Column("updated_at", TypeName = "datetime")]
    public DateTime? UpdatedAt { get; set; }
}